Extension makefile fails to install after complete

Error is
```
/bin/mkdir -p '/usr/share/postgresql-15/extension'
/bin/mkdir -p '/usr/share/postgresql-15/extension'
install -c -m 644 .//bagger_lw_storage.control '/usr/share/postgresql-15/extension/'
install -c -m 644 .//sql/bagger_lw_storage--0.0.1.sql sql/bagger_lw_storage--0.0.1.sql '/usr/share/postgresql-15/extension/'
install: will not overwrite just-created '/usr/share/postgresql-15/extension/bagger_lw_storage--0.0.1.sql' with 'sql/bagger_lw_storage--0.0.1.sql'
make: *** [/usr/lib64/postgresql-15/lib64/pgxs/src/makefiles/pgxs.mk:237: install] Error 1
```

--- a/sql/lenkwerk/storage/Makefile
+++ b/sql/lenkwerk/storage/Makefile
@@ -1,7 +1,6 @@
 EXTENSION = bagger_lw_storage
 EXTVERSION = 0.0.1
 DATA = $(wildcard sql/*--*.sql)
-DATA_built = sql/$(EXTENSION)--$(EXTVERSION).sql
 PG_CONFIG := pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)

